### PR TITLE
Hero section

### PR DESCRIPTION
**Pull request by itsMapleLeaf**
I saw that we were still using `mask-image` on astro.build for the gradient blobs, so I applied the same technique here. LMK if I should do it differently

### CODE REVIEW

File: src/layouts/Base.astro [hunk 1]
```diff
@@ -55,69 +57,68 @@ const pathname = Astro.url.pathname.replace(/\/$/, "")
 
 	<body>
 		<Header />
+		<NoiseContainer>
```
Comment (tony-sull) [inline src/layouts/Base.astro:60]: not really a big deal given the size of this project, but I wonder if there's a way we could avoid all the extra `<div>`s here

Could a component like `<Container noise grid>` wrap it up into a single `<div class="relative">` wrapper?

**Review by tony-sull — COMMENTED** [review]
Looks good!  Added a couple comments, curious if we can simplify noise & grid into a single wrapper?

File: src/layouts/Base.astro (hunk 1 above)
Comment (itsMapleLeaf) [reply src/layouts/Base.astro:60]: Not ideal, the grid is only meant to be applied to the hero section, and the noise applies to the entire thing

Comment (tony-sull) [reply src/layouts/Base.astro:68]: Ah yep, forgot the grid is only part of the viewport!

**Pull request merged**